src/java_bytecode/simple_method_stubbing.h

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************\
+
+Module: Simple Java method stubbing
+
+Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+/// \file
+/// Java simple opaque stub generation
+
+#ifndef CPROVER_JAVA_BYTECODE_SIMPLE_METHOD_STUBBING_H
+#define CPROVER_JAVA_BYTECODE_SIMPLE_METHOD_STUBBING_H
+
+#include <util/irep.h>
+
+class message_handlert;
+class object_factory_parameterst;
```

Comment:
Some compilers insist that this should be a `struct` and thus make the Travis build fail.